app/policies/user_policy.rb

```diff
@@ -44,6 +33,21 @@ def cancel_email_change?
     allow_self_only || edit?
   end
 
+  def has_user_management_permissions
```

Comment:
I think this should be a private method (nitpick too that as it's a question method it should have a question mark suffix)

I'd suggest renaming this as "permissions" is a domain concept.

I actually wonder whether we should leave this as the edit? method and then alias flag_2sv? and reset_2sv? to edit? ?